outlook/mail/main.go

```diff
@@ -64,11 +64,6 @@ func main() {
 			fmt.Printf("failed to send draft: %v\n", err)
 			os.Exit(1)
 		}
-	case "sendMessage":
-		if err := commands.SendMessage(context.Background(), getDraftInfoFromEnv()); err != nil {
-			fmt.Printf("failed to send message: %v\n", err)
-			os.Exit(1)
-		}
```

Comment:
The tool that used this was removed at some point, so I removed it to clean things up.